src/main/java/seedu/address/model/residence/ResidenceAddress.java

```diff
@@ -38,7 +38,7 @@ public static boolean isValidResidenceAddress(String test) {
     }
 
     /**
-     * Returns value of this {@code ResidenceAddress}.
+     * Returns {@code String} value of this {@code ResidenceAddress}.
```

Comment:
similar to above, do we need {@code String}?

maybe can just write Returns string representation of this {@code ResidenceAddress}